libweasyl/libweasyl/models/tables.py

```diff
@@ -219,7 +185,7 @@ def default_fkey(*args, **kwargs):
 favorite = Table(
     'favorite', metadata,
     Column('userid', Integer(), primary_key=True, nullable=False),
```

Comment:
Do we not need an `autoincrement=False` here too? I assumed that the issue was how SQLAlchemy/Alembic was treating Integer() columns set as a primary_key.